utils/config/examples/daos_server_psm2.yml

```diff
@@ -92,3 +85,27 @@ servers:
   # bdev_list: [/tmp/daos-bdev]	# generate nvme.conf as follows:
               # [AIO]
               #   AIO /tmp/aiofile AIO1 4096
+-
```

Comment:
So is the assumption that everyone will want to be running 2 IO servers?

Reply:
that is the intended configuration for a dual socket system which is likely to be the most common target platform. that's probably the best bet for an example configuration IMO